tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -248,9 +248,9 @@ public:
 
 
 /// The container field for an ntuple model, which itself has no physical representation
-class RFieldRoot : public Detail::RFieldBase {
+class RFieldTopLevel : public Detail::RFieldBase {
```

Comment:
It feels awkward to use both "FieldTopLevel" and "TopLevelField" ... unless there is a good reason to.   What am I missing?

Reply:
No concrete reason, this was consistent with other RFieldBase implementors (e.g. `RFieldVector` `RFieldArray`). I can modify those names too if you'd like.

Reply:
I think in this context it is consistent.  We can discuss renaming all the field classes (in a different PR) to `RVectorField`, `RArrayField`, etc.

Reply:
> renaming all the field classes (in a different PR)

Just opened #5831 for this set of changes.

Reply:
I changed things to use `RHeadField` everywhere in a0b6055 
(following the discussion here: https://github.com/root-project/root/pull/5768#discussion_r436809933)